aiida/tools/importexport/dbimport/utils.py

```diff
@@ -241,3 +243,48 @@ def deserialize_field(key, value, fields_info, import_unique_ids_mappings, forei
 
     # else
     return ('{}_id'.format(key), None)
+
+
+def start_header(archive, comment_mode, extras_mode_new, extras_mode_existing, debug):
+    """Print header for import"""
+    import os
+    from aiida.cmdline.utils import echo
+
+    archive = os.path.basename(archive)
+    title = 'IMPORT - !!! DEBUG MODE !!!' if debug else 'IMPORT'
+    result = '\n{}'.format(tabulate([['Archive', archive]], headers=[title, '']))
+
+    parameters = [
+        ['Comment mode', comment_mode],
+        ['New Node Extras mode', extras_mode_new],
+        ['Existing Node Extras mode', extras_mode_existing],
+    ]
+    result += '\n\n{}'.format(tabulate(parameters, headers=['Parameters', '']))
+
+    echo.echo(result)
+
+
+def finish_header(results, import_group_label):
```

Comment:
is this still a "header"?
maybe `def summarize_results(...)`?

Reply:
I don't think any of it is a header in the end - only in the sense that it's above the progress bar, which will always be the case, even for this. I think maybe using "summary" instead of header is indeed more appropriate throughout.